Models/SSBTaxAssessment/GraphQLTypes/TaxAssessmentDatasetType.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Backend.Models.Base.JsonStat;
+using Backend.Models.SSBTaxAssessment.POCO;
+using Backend.utils;
+using GraphQL;
+using GraphQL.Types;
+
+namespace Backend.Models.SSBTaxAssessment.GraphQLTypes
+{
+    public class
+        TaxAssessmentDatasetType : JsonStatDatasetType<TaxAssessment.TaxAssessmentDimension, TaxAssessmentDimensionType>
+    {
+        public TaxAssessmentDatasetType() : base(true)
+        {
+            Field<ListGraphType<LabeledValueTaxType>>("value", "The values for the selected municipalities and years.",
+                null,
+                context =>
+                {
+                    var municipalities = context.Parent.GetArgument<List<string>>("municipalities");
+                    var years = context.Parent.GetArgument<List<string>>("years");
+                    var skipSize = context.Source.Dimension.Size[3];
+                    var municipalityEntrySize = context.Source.Dimension.Size[0];
+                    return (from municipality in municipalities
+                        let municipalityYears = context.Source.Value
+                            .Skip(NorwayTools.MunicipalityCodeToIndexTaxes[municipality] * 294)
+                            .Take(municipalityEntrySize)
+                            .ToList()
+                        select new LabeledValueTax
+                        {
+                            Municipality = NorwayTools.MunicipalityCodeToMunicipalityNameTaxes[municipality],
+                            TaxesForYear = new Collection<TaxesForAGivenYear>(years.Select(year =>
```

Comment:
Perhaps rename it to `AnnualTaxes` or something like that. `TaxesForAGivenYear` reads rather strangely.

Reply:
I agree